Comments/Models/Context.cs

```diff
@@ -113,6 +117,74 @@ public IEnumerable<Location> GetAllCommentsAndQuestionsForDocument(IList<string>
+                                                         && _organisationIDs.Any(o => o.Equals(c.OrganisationId))
+                                                         && !_organisationUserIDs.Contains((int)c.OrganisationUserId)))
+                .IncludeFilter(l => l.Comment.Where(c => c.StatusId == (int)StatusName.SubmittedToLead
+                                                         && _organisationIDs.Any(o => o.Equals(c.OrganisationId))
+                                                         && !_organisationUserIDs.Contains((int)c.OrganisationUserId))
+                    .Select(c => c.SubmissionComment))
+                .IncludeFilter(l => l.Comment.Where(c => c.StatusId == (int)StatusName.SubmittedToLead
+                                                         && _organisationIDs.Any(o => o.Equals(c.OrganisationId))
+                                                         && !_organisationUserIDs.Contains((int)c.OrganisationUserId))
+                    .Select(c=> c.SubmissionComment
+                        .Select(s=> s.Submission)))
+                .IncludeFilter(l => l.Comment.Where(c => c.StatusId == (int)StatusName.SubmittedToLead 
+                                                         && _organisationIDs.Any(o => o.Equals(c.OrganisationId)) 
+                                                         && !_organisationUserIDs.Contains((int)c.OrganisationUserId))
+                    .Select(c=> c.Status))
+                .IncludeFilter(l => l.Comment.Where(c => c.StatusId == (int)StatusName.SubmittedToLead
+                                                         && _organisationIDs.Any(o => o.Equals(c.OrganisationId))
+                                                         && !_organisationUserIDs.Contains((int)c.OrganisationUserId))
+                    .Select(c=> c.OrganisationUser))
+
+                .IncludeFilter(l => l.Question)
+                .IncludeFilter(l => l.Question
+                    .Select(q => q.QuestionType))
+                .IncludeFilter(l => l.Question
+                    .Select(q => q.Answer.Where(a => a.StatusId == (int)StatusName.SubmittedToLead
+                                                     && _organisationIDs.Contains((int)a.OrganisationId) 
+                                                     && !_organisationUserIDs.Contains((int)a.OrganisationUserId))
+                        .OrderByDescending(a=> a.LastModifiedDate).Select(a => a.LastModifiedDate).FirstOrDefault()))
+                .IncludeFilter(l => l.Question
+                    .Select(q => q.Answer.Where(a => a.StatusId == (int)StatusName.SubmittedToLead
+                                                     && _organisationIDs.Contains((int)a.OrganisationId)
+                                                     && !_organisationUserIDs.Contains((int)a.OrganisationUserId))
+                    .Select(a => a.SubmissionAnswer)))
```

Comment:
there's an includefilter to submission from submissioncomment, but not one from submissionanswer.

do you need another includefilter here?

Reply:
I don't seem to need the submissioncomment, or the submission for either so I've removed it.